src/scss/components/ec-grid/ec-grid.story.scss

```diff
@@ -0,0 +1,31 @@
+.grid-story {
```

Comment:
It's not mandatory here but I would follow the conventions we do have for our production scss even here and name the class '.ec-grid-story' and maybe reuse variables, paddings and typography we normally use for our components.